### PR TITLE
Update branding

### PR DESCRIPTION
**Pull request by SandroMachado**
This pull request renames the entire SDK to Uphold.

### CODE REVIEW

**Comment by fixe** [comment]
LGTM. Waiting for organization rename.

**Comment by diogoguimaraes** [comment]
LGTM.

File: README.md [hunk 1]
```diff
@@ -33,25 +33,25 @@ dependencies {
 
 ## Basic usage
 
-In order to learn more about the Bitreserve API, please visit the [developer website](https://developer.bitreserve.org).
+In order to learn more about the Uphold API, please visit the [developer website](https://uphold.com/en/developer).
 
-To use the SDK you must first register an Application and obtain a unique `client_id` and `client_secret` combination. We recommend your first app be [registered in the Sandbox environment](https://sandbox.bitreserve.org/dashboard/profile/applications/developer/new), so you can safely play around during development.
+To use the SDK you must first register an Application and obtain a unique `client_id` and `client_secret` combination. We recommend your first app be [registered in the Sandbox environment](https://sandbox.uphold.org/dashboard/profile/applications/developer/new), so you can safely play around during development.
```
Comment (ruimarinho) [inline README.md]: `uphold.com`

**Comment by ruimarinho** [comment]
LGTM.

**Comment by fixe** [comment]
@SandroMachado please rename GitHub organization.

**Pull request merged**